docs/basic/pgtypes.rst

```diff
@@ -365,3 +365,73 @@ connection or cursor), other connections and cursors will be unaffected::
     ...     "coordinates":[-48.23456,20.12345]}')
     ... """).fetchone()[0]
     '0101000020E61000009279E40F061E48C0F2B0506B9A1F3440'
+
+.. _adapt-enum:
+
+Enum types casting
```

Comment:
"Type casting" is more a psycopg2 term. Please use "Enum adaptation"

Reply:
Fixed.